src/@sdk/helpers/LocalStorageHandler/types.ts

```diff
@@ -124,25 +124,9 @@ export interface IOrderModel {
   number?: string | null;
 }
 
-// export interface IJobsModel {
-//   cart: {
-//     setCartItem?: boolean;
-//   };
-//   checkout: {
-//     setPromoCode?: boolean;
-//   };
-// }
+export type IJobsModel = Record<string, Record<string, boolean>>;
```

Comment:
Why untyping it?
I wonder if it wouldn't be easier if we'd drop one level of nesting, simplifying this type to `Record<"setCartItem" | "setPromoCode", boolean>`

Reply:
Yes, it might be typed, but it is not easy thing to do for me as it is not simple typing - I wrote string, because I didn't want to always redefine this type to be consistent with available methods in job classes.
And I'm not sure if dropping one level would good idea, because, what if we will have the same method name in two different job classes?